content/tracing/setup/javascript.md

```diff
@@ -61,15 +61,10 @@ The following example initializes a Datadog Tracer and creates a Span called `we
 
 ```javascript
 const tracer = require('dd-trace').init()
+const span = tracer.startSpan('web.request')
 
-tracer
-  .trace('web.request', {
-    service: 'my_service'
-  })
-  .then(span => {
-    span.setTag('my_tag', 'my_value')
-    span.finish()
-  })
+span.setTag('my_tag', 'my_value')
```

Comment:
Can we change `my_tag` with something else? for example `http.url` -> `/login` or something more real?